docs/source/release_notes/release-notes.rst

```diff
@@ -111,6 +111,9 @@ develop
           This will provide more accurate metrics for when sweep is doing something.  Sweeping run through the sweep endpoint will now also contribute to these metrics, before it didn't update any metrics which again distorted the view of what work sweep was doing on the DB.
           (`Pull Request <https://github.com/palantir/atlasdb/pull/2535>`__)
 
+    *   - |devbreak|
+        - Simplify and annotate the constructors for SerializableTransactionManager. This should make the code of that class more maintainable.
```

Comment:
If it's a dev break, then the release note should explain why it's a break, and what steps are available to fix it.